best_practices/controllers.rst

```diff
@@ -164,13 +177,14 @@ manually. In our application, we have this situation in ``CommentController``:
 
 .. code-block:: php
 
+    use Doctrine\ORM\EntityManagerInterface;
+
     /**
      * @Route("/comment/{postSlug}/new", name = "comment_new")
      */
-    public function newAction(Request $request, $postSlug)
+    public function newAction(Request $request, $postSlug, EntityManagerInterface $em)
```

Comment:
I've been making this change throughout the docs to avoid `$this->get()` calls in the container. But for the EntityManager specifically, it's not as clear - since `$this->getDoctrine()` is not `$this->get()` (and we're still recommending all other controller shortcuts be used - e.g. `$this->createForm()` instead of injecting the FormFactory).

So, wdyt? It might be safer to change back to preferring `$this->getDoctrine()` throughout the docs?